### PR TITLE
Bump to v4.4.1

### PR DESCRIPTION
**Pull request by ashley-cui**
Release notes + 4.4.1 bump.

```release-note
None

```

### CODE REVIEW

**Comment by ashley-cui** [comment]
@containers/podman-maintainers PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Release Notes
 
+## 4.4.1
+### Changes
+- Added the `podman-systemd.unit` man page, which can also be displayed using `man quadlet` ([#17349](https://github.com/containers/podman/issues/17349)).
+- Documented journald identifiers used in the journald backend for the `podman events` command.
```
Comment (mheon) [inline RELEASE_NOTES.md:6]: For future reference we don't usually talk about documentation in manpages... Not a big deal, but at the same time feel no need to include in the future if it's easier not to

**Comment by mheon** [comment]
LGTM

File: RELEASE_NOTES.md (hunk 1 above)
Comment (ashley-cui) [reply RELEASE_NOTES.md:6]: Ah, I wasn't sure because the authors of the PR's included them in the release notes field, good to know for next time.

**Comment by ashley-cui** [comment]
@containers/podman-maintainers It's green!

**Pull request merged**